.github/workflows/windows.yml

```diff
@@ -55,35 +55,55 @@ jobs:
       run: py -3 -m pip install meson
     - name: test meson
       run: meson -v
-    - name: Cache OpenSSL compilation
-      id: cache-openssl
-      uses: actions/cache@v1
-      env:
-        cache-name: cache-openssl-compilation
-      with:
-        path: openssl
-        key: openssl-1_1_1g
-        restore-keys: openssl-1_1_1g
-    - name: Download openssl source
-      if: steps.cache-openssl.outputs.cache-hit != 'true'
-      run: git clone -b OpenSSL_1_1_1g https://github.com/openssl/openssl
     - uses: ilammy/msvc-dev-cmd@v1
       with:
         arch: x64
     - name: Show from where clang-cl
       run: where clang-cl
-    - name: Configure OpenSSL
-      if: steps.cache-openssl.outputs.cache-hit != 'true'
-      run: perl Configure VC-WIN64A
-      working-directory: .\openssl
-    - name: Compile OpenSSL
-      if: steps.cache-openssl.outputs.cache-hit != 'true'
-      run: nmake
-      working-directory: .\openssl
+    - name: Check if vcpkg is already installed
+      if: steps.cache-vcpkg.outputs.cache-hit == 'true'
+      run: .\vcpkg\vcpkg version
+    - name: Clone vcpkg
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: git clone https://github.com/microsoft/vcpkg
+    - name: Checkout to f4bd6423
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: git checkout f4bd6423
+      working-directory: .\vcpkg
+    - name: Run vcpkg bootstrap
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\bootstrap-vcpkg.bat
+    - name: Install openssl (vcpkg)
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\vcpkg install openssl:x64-windows
+    - name: Install freetype (vcpkg)
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\vcpkg install freetype:x64-windows
+    - name: Install libjpeg (vcpkg)
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\vcpkg install libjpeg-turbo:x64-windows
+    - name: Install lua (vcpkg)
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\vcpkg install lua:x64-windows
+    - name: Install luajit (vcpkg)
+      run: .\vcpkg\vcpkg install luajit:x64-windows
```

Comment:
I think this way luajit will not be cached, right?

Reply:
It will, but I had to ensure it would run at least once.